test_data/windows/basic32.sym

```diff
@@ -1741,7 +1741,7 @@ b723 1 399 56
 b724 1 400 56
 b725 1 401 56
 b726 1 402 56
-FUNC b728 19 0 _EH4_TransferToHandler()
+FUNC b728 19 0 @_EH4_TransferToHandler@8()
```

Comment:
Huh - weird! basic32.pdb contains procedure symbols which have a C decorated name! Not only that, but they also have a non-zero TypeIndex!

I'm guessing that this can only happen in 32 bit.

Reply:
I think this is weird because I was only expecting decorated names in Public symbols, not in Procedure symbols.